MTA-2153: Docs improvements (#839)
windup/windup-documentation: docs/topics/about-rules.adoc

 tag(tags)
----

> Although {ProductShortName} provides a comprehensive set of standard migration rules out-of-the-box, you can create your own custom analyzer rules as well. You can use custom rules to identify use of custom libraries or other components that might not be covered by the provided rulesets.

Reviewer: suggestion (grammar) 
"You can use custom rules to identify _the_ use of custom libraries or other components that _are_ not covered by the provided rulesets."

> Although {ProductShortName} provides a comprehensive set of standard migration rules out-of-the-box, you can create your own custom analyzer rules as well. You can use custom rules to identify use of custom libraries or other components that might not be covered by the provided rulesets.

Author: ```suggestion
Although {ProductShortName} provides a comprehensive set of standard migration rules out-of-the-box, you can create your own custom analyzer rules as well. You can use custom rules to identify the use of custom libraries or other components that are not covered by the provided rulesets.
```
